bzt/modules/python.py

```diff
@@ -614,8 +607,24 @@ def gen_action(self, action_config, indent=None):
                     self.gen_statement("self.assertEqual(%s, _tpl.apply(%r))" %
                                        (tpl % (bys[tag], selector, action), param),
                                        indent=indent))
-            if not action_elements:
-                action_elements.append(self.gen_statement(tpl % (bys[tag], selector, action), indent=indent))
+        elif atype in ('click', 'type', 'keys',
+                       'asserttext', 'assertvalue', 'submit'):
```

Comment:
looks like assert* is kept here by mistake

Reply:
Fixed, thanks!
https://github.com/Blazemeter/taurus/pull/831/commits/e24fa2ce2509edf58eda32e2a1c672432fd5c79f